configs/config.dell_optiplex_9010_sff_uefi

```diff
@@ -0,0 +1,47 @@
+CONFIG_USE_OPTION_TABLE=y
+CONFIG_USE_BLOBS=y
+CONFIG_VENDOR_DELL=y
+CONFIG_BOARD_DELL_OPTIPLEX_9010=y
+CONFIG_ONBOARD_VGA_IS_PRIMARY=y
+# CONFIG_DRIVERS_UART_8250IO is not set
+CONFIG_PCIEXP_CLK_PM=y
+CONFIG_HAVE_EM100_SUPPORT=y
+CONFIG_LOCALVERSION="v0.1.0"
+CONFIG_PAYLOAD_EDK2=y
+CONFIG_EDK2_BOOTSPLASH_FILE="3rdparty/dasharo-blobs/dasharo/evaluation_logo.bmp"
+CONFIG_EDK2_BOOT_MENU_KEY=0x0011
+CONFIG_EDK2_GOP_DRIVER=y
+CONFIG_EDK2_REPOSITORY="https://github.com/Dasharo/edk2.git"
+CONFIG_EDK2_TAG_OR_REV="origin/rebased"
+CONFIG_EDK2_SETUP_MENU_KEY=0x000C
+# CONFIG_EDK2_CBMEM_LOGGING=y
+CONFIG_EDK2_FOLLOW_BGRT_SPEC=y
+CONFIG_EDK2_SD_MMC_TIMEOUT=1000
+CONFIG_TPM_MEASURED_BOOT=y
+CONFIG_VENDOR_DELL=y
+CONFIG_BOOTSPLASH_FILE="dasharo-black.bmp"
+CONFIG_BOOTSPLASH_IMAGE=y
+CONFIG_GENERIC_LINEAR_FRAMEBUFFER=y
+CONFIG_LINEAR_FRAMEBUFFER=y
+CONFIG_TPM2=n
+CONFIG_EDK2_SERIAL_SUPPORT=y
+CONFIG_EDK2_DASHARO_SERIAL_REDIRECTION_DEFAULT_ENABLE=y
+CONFIG_BUILD_IPXE=y
+CONFIG_IPXE_ADD_SCRIPT=y
+CONFIG_IPXE_CUSTOM_BUILD_ID="12345678"
+CONFIG_IPXE_NO_PROMPT=y
+CONFIG_EDK2_ENABLE_IPXE=y
+CONFIG_EDK2_IPXE_OPTION_NAME="Network Boot and Utilities"
+CONFIG_EDK2_DASHARO_SYSTEM_FEATURES=y
+CONFIG_EDK2_DASHARO_SECURITY_OPTIONS=y
+CONFIG_EDK2_DASHARO_USB_CONFIG=y
+CONFIG_EDK2_DASHARO_CHIPSET_CONFIG=y
+CONFIG_EDK2_DASHARO_NETWORK_BOOT_DEFAULT_ENABLE=y
+CONFIG_EDK2_PLATFORMS_REPOSITORY="https://github.com/Dasharo/edk2-platforms"
+CONFIG_EDK2_USE_EDK2_PLATFORMS=y
+CONFIG_EDK2_PLATFORMS_TAG_OR_REV="3323ed481d35096fb6a7eae7b49f35eff00f86cf"
+CONFIG_SMSC_SCH5545_EC_FW_FILE="sch5545_ecfw.bin"
```

Comment:
This assumes the sch5545_ecfw.bin is present in the coreboot directory when building. How are you going to obtain the file? We have no right to distribute it.